server/utils/writeResponse.ts

```diff
@@ -0,0 +1,18 @@
+import { Response } from "express";
+
+interface ResponseData {
+  statusCode?: number;
+  status?: string;
+  data: any;
```

Comment:
how did you use `any` without erros?

Reply:
You should use `unknown` type
I had `never` type but I would like to change this to `unknown`